masterlist.yaml

```
@@ -19084,8 +19084,18 @@ plugins:
         util: '[SSEEdit v4.0.3g](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 2
 
-  - name: '(Serenity|SailorMoonClub|LunarWeapons)\.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/30019/' ]
+  - name: '(Serenity|SailorMoonClub)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/30019/'
+        name: 'Serenity - A Silent Moons Camp Overhaul'
+    tag:
+      - C.Location
+      - Delev
+      - Graphics
+      - Keywords
+      - Names
+      - Stats
+      - Text
```

It's okay to have more than one regex.
So the location info applies to the same files.

```suggestion
  - name: '(Serenity|SailorMoonClub|LunarWeapons)\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/30019/'
        name: 'Serenity - A Silent Moons Camp Overhaul'
  - name: '(Serenity|SailorMoonClub)\.esp'
    tag:
      - C.Location
      - Delev
      - Graphics
      - Keywords
      - Names
      - Stats
      - Text
```